source/blender/makesrna/intern/rna_material.c

```diff
@@ -888,6 +888,15 @@ static void rna_def_material_gamesettings(BlenderRNA *brna)
 		{GEMAT_SHADOW, "SHADOW", 0, "Shadow", "Faces are used for shadow"},
 		{0, NULL, 0, NULL, NULL}
 	};
+
+	static EnumPropertyItem storage_items[] = {
+		{RAS_STORE_AUTO, "AUTO", 0, "Auto Select", "Choose the best supported mode"},
+		{RAS_STORE_VA, "VERTEX_ARRAY", 0, "Vertex Arrays", "Usually the best choice (good performance with display lists)"},
+		{RAS_STORE_VBO, "VERTEX_BUFFER_OBJECT", 0, "Vertex Buffer Objects",
+		                "Typically slower than vertex arrays with display lists, requires at least OpenGL 1.4"},
+		{RAS_STORE_SCENE, "SCENE", 0, "Scene Default", "Use scene storage mode"},
+		{0, NULL, 0, NULL, NULL}
+	};
```

Comment:
Could it be possible to add this info ("This is a problem because some mesh are rendered faster in special storage. e.g geometry instancing meshes with VBO, big meshes with VA+DL and animated meshes with VBO or VA only.") in each text explanation?